logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+
+class IgnoreFilter(logging.Filter):
+    """
+    Customized Log filter that ignores a given message text.
+    """
+
+    def __init__(self, ignored_message: str, name: str = "") -> None:
+        super().__init__(name)
+        self.ignored_message = ignored_message
+        self.enabled = True
+
+    def filter(self, record: logging.LogRecord) -> bool:
+        if not self.enabled:
+            return True
+        return self.ignored_message not in record.getMessage()
+
+    def set_ignored_message(self, message: str) -> None:
+        """Set a new message pattern to ignore."""
+        self.ignored_message = message
+
+    def toggle_filter(self, enable: bool) -> None:
+        """Enable or disable the filter."""
+        self.enabled = enable
+
+    def get_ignored_message(self) -> str:
+        """Get the current ignored message pattern."""
+        return self.ignored_message
+
+
+def set_logger(name: str) -> logging.Logger:
+    logging.basicConfig(level=logging.INFO)
```

Comment:
IMO these fields that we're setting should be from config (or if you think that's overkill, then set via CONSTANTS), not set directly in this function. Should load from config or constants and set in this fn.

Reply:
I've made a minor adjustment to the log level configuration... 
https://github.com/near/gas-station-event-indexer/pull/10/commits/2821db7727595daeda108a29abffddc5ee457072

However doing this properly is going to be a much bigger change.